pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java

```diff
@@ -84,7 +83,6 @@ public void testScanAndFilter() throws Exception {
   }
 
   @Test
-  @Ignore("CALCITE-1751")
   public void testImplWithMultipleFilters() {
```

Comment:
Did the fail reason in CALCITE-1751 has been resolved ? I didn't see any one dug into that , can you give more details ?

Reply:
As described in CALCITE-1751, run those tests in my machine many times, run 'testUserDefinedFunction' many times and it works correctly.

Reply:
Not in your machine, you should validate this on the travis machine.

Reply:
not sure why those tests failed before , run  tests on the travis machine and and it works correctly and stablely.

Reply:
I have no permission for the travis machine, this bug is not stable, it happens after multiple rounds of tests. I have no chance to validate this and there is no evidence that this bug is fixed.

Reply:
Very embarrassed，is there someone can validate this?

Reply:
There two ways to check if the tests are still failing:
* Commit on master and wait the results of Jenkins; if not fixed rollback the commit.
* Create a separate branch, commit there, create a new Jenkins job to follow that branch and check after various runs if everything is good.

The second requires more effort than the time spend for this PR and I don't have much while the first might reduce unnecessary noise to the master.

Reply:
@danny0405 ask in the dev list to get permissions to control the Jenkins jobs. See https://lists.apache.org/thread.html/6a9d873206ca8965d24f91236151f195f4c861aa21aed7dc374b2259@%3Cdev.calcite.apache.org%3E

Reply:
@zabetak Thanks for giving the suggestion, i have sent the mail :)

Reply:
Hey @danny0405, totally unrelated to this bug, but on the subject of email, did you know that @F21 has been trying to contact you via email on behalf of the PMC? Can you please email him or the private list?

Reply:
@julianhyde , oh, sorry for that, i just saw the email, i'm so sorry for the delay @F21 , i'm glad to join you and it's my honor to work with you  !

Reply:
> @julianhyde , oh, sorry for that, i just saw the email, i'm so sorry for the delay @F21 , i'm glad to join you and it's my honor to work with you !

Congratulations！